build.sh

```diff
@@ -1,14 +1,27 @@
-#!sh
+#!/bin/sh
+set -e
 
-cd VirtualBox
+# Script to grab binaries that are going to be bundled with windows installer.
+# Note to maintainers: Update versions used below with newer releases
 
-#http://www.virtualbox.org/manual/ch02.html
-#VirtualBox-4.3.10-93012-Win.exe -extract -silent -path .
-#VirtualBox-4.3.12-93733-Win -extract -silent -path .
-#http://www.catonrug.net/2013/03/virtualbox-silent-install-store-oracle-certificate.html
+(
+	mkdir -p "VirtualBox" && cd "VirtualBox"
+	# http://www.virtualbox.org/manual/ch02.html
 
-cd ../Boot2Docker
+	curl -s -L -o virtualbox.exe http://download.virtualbox.org/virtualbox/4.3.26/VirtualBox-4.3.26-98988-Win.exe
+	virtualbox.exe -extract -silent -path . && rm virtualbox.exe # not neeeded after extraction
```

Comment:
@SvenDowideit @tianon this guy extracts installer for amd64 & x86. do you guys remove x86 one while bundling? if so we can add another rm here.